lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
I didn't propose to make it an external API. Rather a common part for both `toArray` and `toBuffer`.